Source/santactl/SNTCommand.m

```diff
@@ -0,0 +1,47 @@
+/// Copyright 2017 Google Inc. All rights reserved.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+///    Unless required by applicable law or agreed to in writing, software
+///    distributed under the License is distributed on an "AS IS" BASIS,
+///    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+///    See the License for the specific language governing permissions and
+///    limitations under the License.
+
+#import "SNTCommand.h"
+
+@implementation SNTCommand
+
++ (BOOL)requiresRoot { return NO; }
++ (BOOL)requiresDaemonConn { return NO; }
++ (NSString *)shortHelpText { return @""; }
++ (NSString *)longHelpText { return @""; }
+
++ (void)runWithArguments:(NSArray *)arguments daemonConnection:(SNTXPCConnection *)daemonConn {
+  id cmd = [[self alloc] initWithDaemonConnection:daemonConn];
+  [cmd runWithArguments:arguments];
+}
+
+- (instancetype)initWithDaemonConnection:(SNTXPCConnection *)daemonConn {
+  self = [super init];
```

Comment:
nit: Follow the single return format that most other initialization methods implement.

self = [super init];
if (self) {
  _daemonConn = daemonConn;
}
return self;